src/utility/gogoanime.js

```diff
@@ -31,4 +31,98 @@ async function getAnime(slug, episode) {
   };
 }
 
-export default getAnime;
+/**
+ * slug: string
+ * startDtae: anime media.startDate.year
+ * season: anime media.season
+ * @example 'naruto', startDate: '2019', season: 'WINTER'
+ *
+ */
+async function getKitsuEpisodes(slug, startDate, season) {
+  if (!slug || slug === '') return {};
+
+  const newSlug = slug.replace(/[^0-9a-zA-Z]+/g, ' ');
+
+  const findAnime = await scrapeSearch({ keyw: newSlug });
+
+  if (!findAnime || findAnime.length === 0) return {};
+
+  const episodesList = new Map();
+
+  const anime = findAnime[0];
+
+  const gogoEpisodes = (await scrapeAnimeDetails({ id: anime.animeId }))
+    .episodesList;
+
+  let kitsuEpisodes = await fetch('https://kitsu.io/api/graphql', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      query: `query{searchAnimeByTitle(first:5,title:"${newSlug}"){nodes{id season startDate titles{localized}episodes(first:2000){nodes{number titles{canonical}description thumbnail{original{url}}}}}}}`,
```

Comment:
Could you format this into multiple lines to be easier to read?

Reply:
@TomRomeo will do, thanks do pointing it out.